File: P344 Reverse String/Solution.c
char* reverseString(char* s) {
    int len = strlen(s);
    char * result;
    int i;
    int j =0;
    for(i = len-1;i>=0;i--) {
        result[j] = s[i];
        j++;
    }
    result[j] = '\0';
    return result;
}

File: P344 Reverse String/Solution.py
class Solution(object):
    def reverseString(self, s):
        """
        :type s: str
        :rtype: str
        """
        n = len(s)
        result = []
        for i in range(n-1, -1, -1):
            result.append(s[i])
        return ''.join(result)